doc/release/1.5.0-notes.rst

```diff
@@ -24,12 +24,35 @@ For running on PyPy, PyPy3 6.0+ and NumPy 1.15.0 are required.
 Highlights of this release
 --------------------------
 
--
+- wrappers for more than a dozen new ``LAPACK`` routines are now available
+  in `scipy.linalg.lapack`
+- `scipy.optimize.minimize` methods using finite difference numerical 
+  differentiation have gained several new features and there have been
+  performance improvements in `scipy.optimize.linprog` and minimization
```

Comment:
```suggestion
  performance improvements in minimization
```
Thanks, but I don't think those performance enhancements deserve a highlight.  (gh-12043 certainly would, but it looks like that's getting bumped.)